static/css/style.css

```diff
@@ -69,3 +69,67 @@ p, code { line-height:calc(1ex / 0.32); }
   header img { height:4em; padding-bottom:1em; }
   nav li { display:block; }
 }
+
+#bar {
```

Comment:
I'd like to minimize this css and make the style compatible. There should be a dark mode color version.

If it's okay with you, I can suggest commits.